rattle_run.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np 
+import pickle
+import sys
+import os
+import ase.io
+from ocdata.vasp import run_vasp, write_vasp_input_files, VASP_FLAGS
+
+def get_traj_file(pklfile, index):
+    with open(pklfile, 'rb') as f:
+        ans = pickle.load(f)
+        return ans[int(index)]
+
+def get_system_id(trajname):
+    return int(trajname.split("/")[-1].split(".")[0][6:])
+
+
+def main():
+
+    std = float(sys.argv[1]) # 
+    index = sys.argv[2]
+    metafname = "extracted_files_june16_2020.pkl"
+
+    traj_name = "/checkpoint/sidgoyal/electro_done/random1189878.traj" # get_traj_file(metafname, index)
+    opdir = "/checkpoint/sidgoyal/electro_adbulk_rattled_data_round1_june16_2020/"
+
+    full_traj = ase.io.read(traj_name, ":")
+    sys_id = get_system_id(traj_name)
+
+    np.random.seed(sys_id) # seed the sampler 
+    n = len(full_traj)
+    val = int(n * 0.2)
+    assert val > 0
+
+    # sample 20% of frames in the trajectory
+    frame_ids = np.random.choice(n, val, replace=False)
+
+    # set number of outer loop iterations to 1 
+    VASP_FLAGS['nsw'] = 1
```

Comment:
NSW=0 for singlepoints (What I've used for the dataset I've generated thus far)

@ktran9891 confirm?

Reply:
Sorry for super-late response. As per [VASP documentation](https://www.vasp.at/wiki/index.php/NSW), the behavior of `NSW` changes depending on the value of `IBRION`. I'm not entirely sure what value of `IBRION` we used in this context.

I assume everything still worked out?